### DISCUSSION TITLE
Adding ryml to Visual studio project.

### DISCUSSION THREAD

**Discussion by BleblebleU** [biojppm/rapidyaml#111] [Q&A] [answered]
Hello,

I don't know much about CMake or how to use it. Never used the GUI apart from generating visual studio project files.

So can someone please help me get started with how to integrate this library to a visual studio project?
I'm on windows! Any help regarding where and how I can learn to use this would be great, thank you!

**Comment by chetmurphy** [comment, marked as answer]
Use vcpkg manager, search for ryml, and install.  vcpkg is a comman line tool for visual studio and there is a recent package already built. After installing just include the headers and build.

**Comment by biojppm** [comment]
If you have an existing Visual Studio project, then vcpkg is the way to go, as [argued by chetmurphy](https://github.com/biojppm/rapidyaml/discussions/111#discussioncomment-343332).

However, if you are looking to add ryml to an existing cmake project, then [you can find an example here](https://github.com/biojppm/rapidyaml/blob/058e28f895eec239e55324ce0ebe141d5af61080/examples/add_subdirectory/CMakeLists.txt).
